### PR TITLE
Yda-5829 adding documentaiton for troubleshooting tool

### PR DESCRIPTION
**Pull request by FuHsinyu**
Added documentation for the troubleshooting tool.
Should be reviewed together with the troubleshooting scripts

### CODE REVIEW

File: docs/administration/troubleshooting-published-data-packages.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+---
+# How to troubleshoot published data packages
+
+This documentation explains how users can diagnose issues with all existing published data packages using our new troubleshooting tool. The tool performs a series of checks to verify the integrity and compliance of data packages. The scope of this tool includes both data packages that have been successfully published and those that have failed to publish (packages that initiated the publication process but did not succeed). Specifically, it targets data packages with their Attribute-Value Units (AVUs) including `org_publication_status` of `OK`, `Retry`, `Unrecoverable`, or `Unknown`. Note, the `org_` prefix may vary in different Yoda instance as it is a constant variable of `UUORGMETADATAPREFIX` stored in `constant.py`
+
+Alternatively, the tool can diagnose a specific data package when provided with its name.
+
+**Requirement:** Python 3 or higher required, Yoda version 1.10 or later, rodsadmin user
+
+## Check Steps
```
Comment (FuHsinyu) [inline docs/administration/troubleshooting-published-data-packages.md:17]: I added the Check Steps here, but alternatively this can be a part of text in design section not the administration section. In the end, I thought this can help administrator understands the check steps, and it reveals what the cmd will do specifically.  so I put it here. Other remarks / suggestions are welcomed

File: docs/administration/troubleshooting-published-data-packages.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+---
+parent: Administration Tasks
+title: Troubleshooting published data packages
+nav_order: 21
+---
+# How to troubleshoot published data packages
+
+This documentation explains how users can diagnose issues with all existing published data packages using our new troubleshooting tool. The tool performs a series of checks to verify the integrity and compliance of data packages. The scope of this tool includes both data packages that have been successfully published and those that have failed to publish (packages that initiated the publication process but did not succeed). Specifically, it targets data packages with their Attribute-Value Units (AVUs) including `org_publication_status` of `OK`, `Retry`, `Unrecoverable`, or `Unknown`. Note, the `org_` prefix may vary in different Yoda instance as it is a constant variable of `UUORGMETADATAPREFIX` stored in `constant.py`
```
Comment (claravox) [inline docs/administration/troubleshooting-published-data-packages.md]: Not sure if anyone actually changes the org_ prefix. Maybe reword as:
```suggestion
This documentation explains how users can diagnose issues with all existing published data packages using our new troubleshooting tool. The tool performs a series of checks to verify the integrity and compliance of data packages. The scope of this tool includes both data packages that have been successfully published and those that have failed to publish (packages that initiated the publication process but did not succeed). Specifically, it targets data packages with their Attribute-Value Units (AVUs) including `org_publication_status` of `OK`, `Retry`, `Unrecoverable`, or `Unknown`. Note, the `org_` prefix is defined by the constant variable of `UUORGMETADATAPREFIX` stored in `constants.py` in the ruleset.
```

File: docs/administration/troubleshooting-published-data-packages.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+### 2. Specific Package Check
+
+To inspect a single data package:
+
+```bash
+python3 troubleshoot-published-data.py -p <package-name>
+```
+
+An example of data package name is `research-core-0[1722266819]`
```
Comment (claravox) [inline docs/administration/troubleshooting-published-data-packages.md]:
```suggestion
The package name can be specified as the ending to the path of the data package (example: `research-core-0[1722266819]`) or the whole path (example: `/tempZone/home/vault-default-3/research-default-3[1728048679]`).
```

File: docs/administration/troubleshooting-published-data-packages.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+### 3. Log results and offline mode
+
+By default, the results are displayed to terminal (stdout). Furthermore, to save the detailed output to a log file execute:
+
+```bash
+python3 troubleshoot-published-data.py -l -o
+```
+
+- The -l option enables logging mode. Saving log to `/var/lib/irods/log/troubleshoot_publications.log`
+- The -o option enables offline mode, which skips several tests related to connecting to remote server.
```
Comment (claravox) [inline docs/administration/troubleshooting-published-data-packages.md]:
```suggestion
- The -o option enables offline mode, which skips several tests related to connecting to remote servers. This is useful when testing on a local development environment.
- The -n option enables no datacite mode, which skips the datacite checks. This is also useful when testing on a local development environment.
```

File: docs/administration/troubleshooting-published-data-packages.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+This step compares the contents of the local landing page file with the remote landing page to ensure they match. It does this by sending a URL request to download the HTML of the data package's landing page and comparing it with the local HTML file. Note that if there is no internet connection, you should enable the `offline` mode. In offline mode, this step checks if the local landing page file exists but does not verify the correctness of its content.
+
+### Combined JSON Integrity
+
+This step checks the integrity of the combined JSON file by verifying its URL online and confirming the existence of the file. It accomplishes this by checking if the metadata JSON sent to OAI-PMH server can be found in the OAI-PMH repository. In offline mode, it only checks whether package's `-combi.json` file exists locally
+
+
+## Commands
+
+The tool can be used with various options as detailed below.
```
Comment (claravox) [inline docs/administration/troubleshooting-published-data-packages.md]: We either need to explain here that you should cd to the tools directory first before running these commands, or provide the absolute path to the script in all these examples.

**Review by claravox — COMMENTED** [review]
Looks good to me!

**Review by lwesterhof — APPROVED** [review]
Great!
